certif/tests/unit/components/new-certification-candidate-modal_test.js

```diff
@@ -306,5 +316,26 @@ module('Unit | Component | new-certification-candidate-modal', function (hooks)
       // then
       assert.deepEqual(modal.args.candidateData.complementaryCertification, secondComplementaryCertification);
     });
+
+    test('it should remove the complementary when no complementary is selected', function (assert) {
+      // given
+      modal.args.candidateData = {
+        complementaryCertification: {
+          id: 0,
+          label: 'Certif complémentaire 1',
+          key: 'COMP_0',
+        },
+      };
+
+      const noneChoice = {
+        target: { value: { label: 'none' } },
```

Comment:
[Value ne contient pas de sous-propriété label](https://html.spec.whatwg.org/multipage/input.html#dom-input-value), on le met direct à none et il prend cette valeur

```suggestion
        target: { value: 'none' },
```